data/huion-h640p.tablet

```diff
@@ -0,0 +1,36 @@
+# HUION
+# H640P
+# 
+#      *-----------------------------------*
+#    A |                                   |
+#    B |                                   |
+#    C |                                   |
+#      |              DISPLAY              |
+#    D |                                   |
+#    E |                                   |
+#    F |                                   |
+#      *-----------------------------------*
+#
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/145
+
+[Device]
+Name=Huion H640P
+ModelName=
+DeviceMatch=usb:256c:006d
+PairedIDs=usb:256c:006d
+Class=Bamboo
+Width=6.3
+Height=3.9
```

Comment:
these are supposed to be the marketing numbers (no-one uses them anyway afaik), so integers only please. `6x4` I guess for this one, unless marketing was overly generous and describes it as `7x4`.